_posts/2021-06-25-feature-highlight-reporting.markdown

```diff
@@ -0,0 +1,79 @@
+
+## Quick Start User Guide
+
+### Installation
+
+Reporting comes packaged in with the OpenSearch and OpenSearch Dashboards downloads. You can follow the installation instructions on [opensearch.org/downloads](http://opensearch.org/downloads) 
+
+### How do I create a report?
+
+**Visual reports** are created inside the Visualization or Dashboards experience. 
+
+![visual reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/visual-reports.gif){: .img-fluid}
+
+**CSV reports** are created from the Discover tab.
+
+![csv reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/csv-reports.gif){: .img-fluid}
+
+**Scheduled reports** are configured from inside the Reporting UI under OpenSearch Plugins.
+
+![scheduled reports](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/create-report-definition.gif){: .img-fluid}
+
+## Development internals
+
+The reporting feature uses a custom [minimal build](https://github.com/opensearch-project/dashboards-reports/tree/main/dashboards-reports/rendering-engine/headless-chrome) of headless-chromium. Chromium is used as the rendering engine to generate visual reports. Below is are multiple flow diagrams for the various actions of Reporting. 
+
+**Creating a report definition**: When a user defines a report in OpenSearch Dashboards, the Reporting Dashboards plugin saves that definition in an index in OpenSearch. 
+
+**Downloading a report:** When a user downloads a report, the OpenSearch Dashboards Reporting plugin fetches the data from OpenSearch. After the data is fetched, the report is generated and downloaded to the users client.
+
+**Running a reporting job:** When a schedule report begins, data is fetched from OpenSearch, next the OpenSearch Dashboards Reporting plugin generates the report. Once the report is generated, it is sent to the configured destination in the reporting policy and the job status is updated and stored back in OpenSearch.
+
+![flow diagram](/assets/media/blog-images/2021-06-25-feature-highlight-reporting/flow-diagram.png){: .img-fluid}
+
+## Community contributions
+
+Reporting was built by multiple contributors from the community (some PRs listed below). We presented the initial Reporting contributions in a community meeting with the [Reports RFC](https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/master/docs/dev/Kibana-Reporting-Design-Proposal.md). Our team went over the initial design and requirements, and our high level plan to develop the feature. After the community meeting, some engineers at Dassault Systems expressed interest in contributing and helped build the CSV functionality. Since the initial release, there have been numerous other community contributions to improve and customize the project (see below). We thank all the contributors who have helped build the Reporting feature!
+
+* [50](https://github.com/opendistro-for-elasticsearch/kibana-reports/pull/50)
```

Comment:
A list of numbers looks weird - can we also include the type of the PRs?

Reply:
fixed